Makefile

```diff
@@ -4,7 +4,7 @@
 all: test
 
 cover:
-	go test -cover -tags=integration
+	go test -cover -tags=integration -coverprofile=coverage.out
```

Comment:
Can we remove this? I'll update the Makefile and `make.cmd` so those flags will be changeable by the user.